tekton/release-cheat-sheet.md

```diff
@@ -1,121 +1,170 @@
 
-    **If you are backporting fixes to an older release, include this flag: `--param=releaseAsLatest="false"`**
+1. Once the pipeline run is complete, check its results:
 
-    ```bash
-      tkn pipeline start \
-        --param=versionTag=${VERSION_TAG} \
-        --serviceaccount=release-right-meow \
-        --resource=source-repo=${GIT_RESOURCE_NAME} \
-        --resource=bucket=tekton-triggers-bucket \
-        --resource=builtEventListenerSinkImage=event-listener-sink-image \
-        --resource=builtControllerImage=triggers-controller-image \
-        --resource=builtWebhookImage=triggers-webhook-image \
-        --resource=builtCoreInterceptorsImage=core-interceptors-image \
-        --resource=notification=post-release-trigger \
-        triggers-release
-    ```
+   ```bash
+   tkn --context dogfooding pr describe <pipeline-run-name>
 
-8. Watch logs of triggers-release.
+   (...)
+   📝 Results
 
-9. The YAMLs are now released! Anyone installing Tekton Triggers will now get the new version. Time to create a new GitHub release announcement:
+   NAME                    VALUE
+   ∙ commit-sha            ff6d7abebde12460aecd061ab0f6fd21053ba8a7
+   ∙ release-file           https://storage.googleapis.com/tekton-releases-nightly/triggers/previous/v20210223-xyzxyz/release.yaml
+   ∙ release-file-no-tag    https://storage.googleapis.com/tekton-releases-nightly/triggers/previous/v20210223-xyzxyz/release.notag.yaml
 
-    1. Run Draft Release Task.
-      ```bash
-      tkn task start \
-        -i source=${GIT_RESOURCE_NAME} \
-        -p package=tektoncd/triggers \
-        -p release-tag=${VERSION_TAG} \
-        -p previous-release-tag=${PREVIOUS_VERSION_TAG} \
-        create-draft-triggers-release
-      ```
+   (...)
+   ```
 
-    1. Watch logs of create-draft-release TaskRun for errors.
+   The `commit-sha` should match `$TEKTON_RELEASE_GIT_SHA`.
+   The two URLs can be opened in the browser or via `curl` to download the release manifests.
 
-        ```bash
-        tkn tr logs -f create-draft-release-run-# this will end with a random string of characters
-        ```
+    1. The YAMLs are now released! Anyone installing Tekton Triggers will now get the new version. Time to create a new GitHub release announcement:
 
-    1. On successful completion, a URL will be logged. Visit that URL and sort the
-    release notes. **Double-check that the list of commits here matches your expectations
-    for the release.** You might need to remove incorrect commits or copy/paste commits
-    from the release branch. Refer to previous releases to confirm the expected format.
+    1. Create additional environment variables
 
-    1. Publish the GitHub release once all notes are correct and in order.
+    ```bash
+    TRIGGERS_OLD_VERSION=# Example: v0.11.1
+    TEKTON_PACKAGE=tektoncd/triggers
+    ```
 
-10. Edit `README.md` on `master` branch, add entry to docs table with latest release links.
+    1. Create a `PipelineResource` of type `git`
```

Comment:
Ahh the crease draft release still requires a git resource...Still this is much better than what we had before.